JoinColony/colonyDapp
# Connect "Transfer funds" transaction

To test:

- Run `npm run dev:heavy -- --skip-webpack`
- Follow the instructions in https://github.com/JoinColony/colonyDapp/pull/3765 to get the foreign bridge running. Make sure you have the latest changes from `maint/cross-chain-safe`.
- Add the env variable values to your .env and then start webpack (`npm start webpack` in `/colonyDapp`).
- Add a safe with balances. For example: `0x4cb09344de5bCCD45F045c5Defa0E0452869FF0f` (Rinkeby)
- Go to safe control, select "transfer funds", and fill the form. The token selector works as a balance selector.
- Confirm the transaction.
- In the terminal window in which you're running the foreign bridge, you should see a confirmation that the funds have been transferred out of the safe. In the console in the browser, you should see confirmation that the bridging transaction was successful.

NOTE: In the local env, it's not expected that the safe you added is used to transfer the funds. A local safe with 100 test tokens in it is going to be used.

Resolves #3720
